R/gwlZypTest.R

```diff
@@ -11,22 +11,27 @@
 # See the License for the specific language governing permissions and limitations under the License.
 
 #' Perform Mann-Kendall trend test on many wells
-#'
-#' Uses the zyp package to calculate Mann-Kendall trend test on pre-whitened
+#' 
+#' Uses the zyp package to calculate Mann-Kendall trend test on pre-whitened 
 #' data (to remove autocorrelation) for many wells, using one or both of two 
 #' Pre-whitening methods, see zyp documentation
-#' @import zyp
-#' @param dataframe dataframe containing: an ID column (specified in \code{by}) and a column of values
-#' @param  wells vector of well numbers to test. Default NULL does all in dataframe
-#' @param  byID the name of the ID column
-#' @param  col the name of the column with the GWL values
-#' @param  method "both" (default), "yuepilon", or "zhang"
-#' @export
+#' 
+#' @param dataframe dataframe containing: an ID column (specified in \code{by})
+#'   and a column of values
+#' @param wells vector of well numbers to test. Default NULL does all in
+#'   dataframe
+#' @param byID the name of the ID column
+#' @param col the name of the column with the GWL values
+#' @param method "both" (default), "yuepilon", or "zhang"
+
 #' @return a dataframe of results for all wells evaluated
+
 #' @examples \dontrun{
 #'
 #'}
-gwlZypTest <- function(dataframe, wells=NULL, byID, col, method="both") {
+#' @export
+
+gwlZypTest <- function(dataframe, wells = NULL, byID, col, method = "both") {
   
```

Comment:
We should check the existence of columns named in `byID` and `col` - this isn't really in scope for you, so I will open an issue.

Reply:
#12 - probably should look across the board to make sure it's done consistently (for later)